### ISSUE TITLE
Unable to compile on MacOS M1 after upgrading to Nix 23.11

### ISSUE THREAD

**Issue by mirosval** [symphorien/nix-du#20]
Worked previously, I just upgraded to 23.11

```shell
error: builder for '/nix/store/5ya9w2x6bh3f96mwynkb3hhikw4g0ns7-nix-du-1.1.1.drv' failed with exit code 101;
       last 10 log lines:
       >                 populateGraph::$_0::operator()() const::'lambda'(nix::StorePath const&)::operator()(nix::StorePath const&) const in libnix_adapter.a(wrapper.o)
       >                 std::__1::pair<std::__1::__tree_const_iterator<nix::StorePath, std::__1::__tree_node<nix::StorePath, void*>*, long>, std::__1::back_insert_iterator<std::__1::vector<nix::StorePath, std::__1::allocator<nix::StorePath>>>> std::__1::__copy_loop<std::__1::_ClassicAlgPolicy>::operator()[abi:v160006]<std::__1::__tree_const_iterator<nix::StorePath, std::__1::__tree_node<nix::StorePath, void*>*, long>, std::__1::__tree_const_iterator<nix::StorePath, std::__1::__tree_node<nix::StorePath, void*>*, long>, std::__1::back_insert_iterator<std::__1::vector<nix::StorePath, std::__1::allocator<nix::StorePath>>>>(std::__1::__tree_const_iterator<nix::StorePath, std::__1::__tree_node<nix::StorePath, void*>*, long>, std::__1::__tree_const_iterator<nix::StorePath, std::__1::__tree_node<nix::StorePath, void*>*, long>, std::__1::back_insert_iterator<std::__1::vector<nix::StorePath, std::__1::allocator<nix::StorePath>>>) const in libnix_adapter.a(wrapper.o)
       >                 boost::io::basic_altstringbuf<char, std::__1::char_traits<char>, std::__1::allocator<char>>::overflow(int) in libnix_adapter.a(wrapper.o)
       >                 std::__1::vector<boost::io::detail::format_item<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<boost::io::detail::format_item<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>::assign(unsigned long, boost::io::detail::format_item<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&) in libnix_adapter.a(wrapper.o)
       >                 ...
       >           ld: symbol(s) not found for architecture arm64
       >           clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
       >
       >
       > error: could not compile `nix-du` (bin "nix-du") due to previous error
       For full logs, run 'nix log /nix/store/5ya9w2x6bh3f96mwynkb3hhikw4g0ns7-nix-du-1.1.1.drv'.
```

Full Log here:
[nix-du.log](https://github.com/symphorien/nix-du/files/13625354/nix-du.log)

**Comment by symphorien** [comment]
can you try if the issue is fixed in branch https://github.com/symphorien/nix-du/tree/darwin_build_llvm_bump ?

**Comment by symphorien** [comment]
fixed in v1.2.0
